css-positioning/absolute.css

```diff
@@ -0,0 +1,53 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+  position: relative;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 20px;
+  bottom: 500px;
+  left: 240px;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 165px;
+  right: 975px;
+  font-weight: bold;
+  bottom: 1750px;
+  font-stretch: narrow;
+  left: 15px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
+  /* your code here */
+  position: absolute;
+  bottom: 175px;
+  top: 8px;
```

Comment:
If you fix the issue with the `meme` class I mentioned above, this shouldn't be necessary anymore